### PR TITLE
[BACKLOG-6738] Icon in fieldhelp no longer disappears when a user hov…

### PR DESCRIPTION
**Pull request by DFieldFL**
…ering over it for IE10

### CODE REVIEW

**Comment by DFieldFL** [comment]
@pminutillo @rfellows please review and merge

File: pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/crystal/globalCrystal.css [hunk 1]
```diff
@@ -5542,3 +5534,11 @@ treecontrol.tree-classic li .tree-selected {
 #sidepanel #sidepaneltabset > div[tabid=generaltab] > span{
   max-width:62px;
 }
+
```
Comment (rfellows) [inline pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/crystal/globalCrystal.css]: i don't see a change other than moving these lines down in the file... am i blind?

**Comment by DFieldFL** [comment]
Yep that is it. There was forward and a backward compatibility issue so moving it down fix it.

**Comment by rfellows** [comment]
WOW, that is crazy

**Pull request merged**